### PR TITLE
HARVESTER: use runStrategy replace running

### PR DESCRIPTION
**Pull request by WuJun2016**
# links
- https://github.com/harvester/harvester/issues/2060

### CODE REVIEW

File: assets/translations/en-us.yaml [hunk 1]
```diff
@@ -5557,6 +5557,7 @@ harvester:
         Detach Volume
     restartTip: Restart the virtual machine now to take effect of the configuration changes.
     createRunning: Start virtual machine on creation
+    runStrategy: RunStrategy
```
Comment (n313893254) [inline assets/translations/en-us.yaml:5560]:
```suggestion
    runStrategy: Run Strategy
```

**Review by n313893254 — CHANGES_REQUESTED** [review]
Minor typo. others LGTM

File: assets/translations/en-us.yaml (hunk 1 above)
Comment (WuJun2016) [reply assets/translations/en-us.yaml]: Done,  PTAL. thanks

**Review by n313893254 — APPROVED** [review]
LGTM

**Pull request merged**